new_lineage_panel/src/Lineage.tsx

```diff
@@ -335,20 +337,60 @@ export const Lineage = () => {
   useEffect(() => {
     const _flow = flow.current;
     if (!_flow) return;
-    const _edges = _flow.getEdges();
-    if ((selectCheck && nonSelectCheck) || (!selectCheck && !nonSelectCheck)) {
-      for (const e of _edges) e.hidden = false;
+    (async () => {
```

Comment:
any reason for doing this self invoking function? can this be done without this?